SPECS/dracut/dracut.signatures.json

```
@@ -1,7 +1,18 @@
 {
  "Signatures": {
-  "dracut-055.tar.xz": "4baa08206cceeb124dbf1075a0daf774b5a8f144ce2e01d82a144af3020fd65b",
-  "lgpl-2.1.txt": "dc626520dcd53a22f727af3ee42c770e56c97a64fe3adb063799d8ab032fe551",
+  "0001-Add-mkinitrd-support-to-dracut.patch": "8202173183bec85e2286890b743665d40570ed1c6343fe4ed15df0359a71524c",
+  "0002-disable-xattr.patch": "cc17cae2e63ad13353ad4ccdbf6a9658a6bb042055c274b1c2f569588812eea9",
+  "0003-fix-initrd-naming-for-photon.patch": "ce63bd915f85c4fc4e33582357a35bd5169b3983e3e6445cae77748af6af1388",
+  "0004-fix-hostonly.patch": "4f7e011eed3f5b6af3ac507702ec923f10080a3465ebd7f859cbf550e35e9db6",
+  "0005-mkinitrd-verbose-fix.patch": "93c7fdfa07c7cf4cb9912d5f9b88f74d0e97f2216d00a61f7ea0df2ee03cfc93",
+  "0006-dracut.sh-validate-instmods-calls.patch": "65b5ea2bd39c80c7e8bdd755707a9b36e2f27046b39834f0af5606f2d0361d94",
+  "0007-feat-dracut.sh-support-multiple-config-dirs.patch": "97b8bcf0ae4d85a3c356c09c8b06241d05a9ab75c5ff1dbb0bb4141237925f82",
+  "0008-fix-dracut-systemd-rootfs-generator-cannot-write-out.patch": "ee2f375aa273b56d63eb5b0269a765fcb34506b512e04303602539341f317527",
+  "0009-install-systemd-executor.patch": "44c8730c1182e099059ecfe60867d89c66d70a7202509b4b375edb0922bffbd8",
+  "20overlayfs": "",
+  "allow-liveos-overlay-no-user-confirmation-prompt.patch": "a98c93e031517e40d3de7f8b75980e07024dff026624122d3a2b623c79cb942e",
+  "dracut-059.tar.gz": "eabf0bb685420c1e1d5475b6855ef787104508f0135ff570312845256e0fcecf",
+  "fix-functions-Avoid-calling-grep-with-PCRE-P.patch": "47165eb50a8c46b621605958e67d9ade26554fe5408ec426e5871ca000db6c25",
```

Generally we don't add patch files to the signatures.json since they tend to change a lot and are tracked by git already (and the tools won't download them if they are missing). Local source files are a grey area since they are also tracked by bit, but to simplify handling in the tools we treat them the same as remote sources.
```suggestion
  "20overlayfs": "",
  "allow-liveos-overlay-no-user-confirmation-prompt.patch": "a98c93e031517e40d3de7f8b75980e07024dff026624122d3a2b623c79cb942e",
  "dracut-059.tar.gz": "eabf0bb685420c1e1d5475b6855ef787104508f0135ff570312845256e0fcecf",
  "fix-functions-Avoid-calling-grep-with-PCRE-P.patch": "47165eb50a8c46b621605958e67d9ade26554fe5408ec426e5871ca000db6c25",
```

---

ah ok sorry i have no idea how the 'signatures' file works, since *all* the files (except the tarball that's in our blobstore) are in git and so shouldn't actually be in the 'signatures' file.

---

updated signatures file